signer/src/stacks/api.rs

```diff
@@ -80,6 +74,47 @@ const DUMMY_STX_TRANSFER_PAYLOAD: TransactionPayload = TransactionPayload::Token
     TokenTransferMemo([0; 34]),
 );
 
+/// The names of all the read-only functions used in the signers for any of
+/// the sbtc smart contracts.
+#[derive(Clone, Copy, Debug, Hash, PartialEq, Eq, strum::Display, strum::IntoStaticStr)]
+#[strum(serialize_all = "kebab-case")]
```

Comment:
While functionally equivalent, I'm wondering if we should opt to use an explicit `serialize` (or `to_string`) `strum` attribute on each variant, for a more "obvious" mapping of the variant to the method called; also for `ClarityMapName` and `DataVarName`.

Reply:
Yeah, I agree. I did it like how it is now, changed it to how you're suggesting, and then changed it back. I might have done that two times 😞. I think I'm going to move away from the `enum` though.